src/main/java/org/folio/orders/utils/HelperUtils.java

```diff
@@ -342,6 +343,16 @@ public static String getEndpointWithQuery(String query, Logger logger) {
       errors.add(ErrorCodes.ELECTRONIC_LOC_QTY_EXCEEDS_COST);
     }
 
+    if (compPOL.getPhysical().getCreateInventory() == Physical.CreateInventory.INSTANCE_HOLDING_ITEM
```

Comment:
Instead of repeating these blocks of code, can we pull this into a shared/common `validateMaterialTypes(...)` method, a la `validateCostPrices(...)`?